cmd/kwild/config/config.go

```diff
@@ -555,6 +557,8 @@ func DefaultConfig() *KwildConfig {
 			DBPort:               "5432", // ignored with unix socket, but applies if IP used for DBHost
 			DBUser:               "kwild",
 			DBName:               "kwild",
+			RPCTimeout:           Duration(45 * time.Second),
+			ReadTxTimeout:        Duration(5 * time.Second),
```

Comment:
I was very surprised to find that the default for the db read timeout was 100ms on v0.6.x.  Isn't that extremely short?

Also note that the server timeout is separate, much longer, applies to all requests, and is primarily a defensive mechanism for the server.